recipes/autogrid/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "autogrid" %}
+{% set version = "4.2.9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ccsb-scripps/AutoGrid/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: d48eb3c72eb3708016ee27c297e2b7c3ad1e3d10e3ea2b0ea0cba95d4dcfbaf7
+
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - make
+    - tcsh
+    - meson
+    - libgomp
+  host:
+    - libgcc
+    - libstdcxx
+    - libgfortran
+    - libgomp
+  run:
+    - libgcc
+    - libstdcxx
+    - libgfortran
+    - libgomp
+
```

Comment:
```suggestion
```

Reply:
You shouldn't need to manuallyu specify these libs because they should be exported by packages in the build requirements such as the compiler packages.